doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,840 @@
+.. _ncs_release_notes_190:
+
+|NCS| v1.9.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF21, nRF52, nRF53, and nRF91 series.
+The SDK includes open source projects (TF-M, MCUboot OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and re-distributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+The feature is made available in its current state though the design and interfaces can change between release tags.
+The feature will also be labelled with "EXPERIMENTAL" in Kconfig files to indicate this status.
+Build warnings will be generated to indicate when features labelled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+
+Highlights
+**********
+
+* Added experimental support for Nordic Distance Toolbox with examples. The software package adds phase-based distance measurement support for the nRF52 and nRF53 Series of devices.
+* Added support for Amazon Frustration-Free Setup (FFS) for Matter devices.
+* Added deep power-down mode to QSPI driver to optimize energy usage for solutions using an external flash memory.
+* Added support for micro:bit v2 board.
+* Added support for filtered ephemeris that improves A-GPS download size.
+* Updated the support for Periodic Advertising with the SoftDevice Controller. It is no longer experimental.
+* Updated the Zigbee solution on nRF5340. It is no longer experimental and is now Zigbee Platform Certification compliant.
+* Replaced proprietary sockets with nrf_modem APIs that include link-time optimization, thereby reducing memory usage.
+
```

Comment:
Please add highlight for nrf_security:

* Added experimental support for PSA crypto drivers in nrf_security
* Added PSA crypto drivers for CryptoCell and Oberon

Can be made into one line as well

Reply:
Moving to the sdk-nrfxlib section as agreed offline.